frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -91,11 +101,25 @@ export const VMResourceSummary: React.FC<VMResourceSummaryProps> = ({
   kindObj,
 }) => {
   const { t } = useTranslation();
+  const templatesResource: WatchK8sResource = {
```

Comment:
We should load the templates together with user templates at `vm-details-page`

Reply:
removed the `WatchK8sResource` hook and now using the firehose mechanism for both `templats` and `commonTemplates`